apps/webapp/src/components/Nav/main-nav.tsx

```
@@ -2,127 +2,74 @@
+        <div
+          key={item.href}
+          onClick={() => onLinkClick(item.href)}
+          className={cn(
+            "flex items-center rounded-md px-3 py-2 text-sm font-medium cursor-pointer",
+            "transition-colors",
+            "hover:bg-accent hover:text-accent-foreground",
+            pathname === `/${item.href}`
+              ? "bg-accent text-accent-foreground"
+              : "text-muted-foreground hover:text-primary"
+          )}
+        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)